README.md

```diff
@@ -402,12 +402,15 @@ window._zx.push({"id": "25GHTE9A07DF67DFG90T"});
 
 #### Conversion tracking
 
+Lead events and sale events use similar text snippets, but different url paths. For example:
+`https://foo.zanox.com/abc/?123456789`, where `abc` is the changing url path. This can be set dynamically along with the rest of the parameters by setting the `path_extension`. 
+
 This is an example of a lead event:
 
 ```
 def show
   tracker do |t|
-    t.zanox :track, { order_i_d: 'DEFC-4321'}
+    t.zanox :track, { order_i_d: 'DEFC-4321', path_extension: 'abc' }
```

Comment:
for lead events it's `ppl`, and for sale events it's `pps`. 
Let me see if I can do this differently so that it doesn't have to be set every time